### ISSUE TITLE
Tone.Volume: mute ignored on volume change

### ISSUE THREAD

**Issue by timpulver** [Tonejs/Tone.js#221]
When muting a volume node and afterwards setting the volume, mute is ignored.
```javascript
var volume = new Tone.Volume(); 
volume.set("mute", true);
volume.set("volume", -6); // now signal isn’t muted any more
```

If this is expected bahavior it would be good to mention it in the docs. Optimally the Volume-node would not set the volume in the internal audio node, store the value internally and set the stored value on unmute.

**Comment by tambien** [comment]
Thanks for the concise example. I prefer to avoid hidden state where possible. I think that the volume should change from muted when it is set, similar to how `volume.value` is `-Infinity` when it is muted.

I updated Tone.Volume so that `mute` reflects the fact that it is no longer muted when the volume is set (before mute was still returning `true` even if the volume was changed like in your example).
